### DISCUSSION TITLE
Expired SSL certificate on GitHub pages

### DISCUSSION THREAD

**Discussion by twiecki** [community/community#23572] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>My personal blog <a href="http://www.twiecki.io" rel="noopener nofollow ugc">www.twiecki.io</a> (GH pages link: <a href="http://twiecki.github.io" rel="noopener nofollow ugc">twiecki.github.io</a>) has an expired SSL certificate. I thought it would get updated automatically but it’s been over 1 week.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/6/a63facec7be9a0cdf766fa472546189e1a53e7fc.jpeg" data-download-href="/uploads/short-url/nIHGq1jK6zTt2Aebsckb6tkXZdq.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/674200/181077967-cfc85cd5-b7ef-4836-89fa-680c40805a47.jpeg" alt="image" data-base62-sha1="nIHGq1jK6zTt2Aebsckb6tkXZdq" width="690" height="245" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/6/a63facec7be9a0cdf766fa472546189e1a53e7fc_2_690x245.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/a/6/a63facec7be9a0cdf766fa472546189e1a53e7fc_2_1035x367.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/6/a63facec7be9a0cdf766fa472546189e1a53e7fc_2_1380x490.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/6/a63facec7be9a0cdf766fa472546189e1a53e7fc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2070×736 82.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Any ideas?</p>
<p>Best,<br>
Thomas</p>

**Comment by yoannchaudet** [comment]
<p>Hello,</p>
<p>A DNS change triggered a certificate renewal attempt for your site.</p>
<p>In your case, it seems an invalid IP address has been added to your domain: <code>162.255.119.56</code>.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/b/0b2e97f91d1efaef70c790ce31de60d9353d90e1.png" data-download-href="/uploads/short-url/1AV4S8icQkLUdA1k0Brx7LPjZ3X.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/14911070/181077970-65ac8111-52b8-4e5e-ac50-2293c50494c0.png" alt="image" data-base62-sha1="1AV4S8icQkLUdA1k0Brx7LPjZ3X" width="690" height="436" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0b2e97f91d1efaef70c790ce31de60d9353d90e1_2_690x436.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0b2e97f91d1efaef70c790ce31de60d9353d90e1_2_1035x654.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/b/0b2e97f91d1efaef70c790ce31de60d9353d90e1.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0b2e97f91d1efaef70c790ce31de60d9353d90e1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1270×804 86.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>This looks to be from Namecheap and it has been causing few issues recently. It could have been added if you tried to setup some HTTP redirections there.</p>
<p>Removing the invalid IP should clear the situation and help with the certificate renewal.</p>

**Comment by twiecki** [comment]
<p>Thanks <a class="mention" href="/yoannchaudet">@yoannchaudet</a>. However, I don’t see a redirect or that IP somewhere (I posted the screenshot from namecheap above). Do you see anything wrong there?</p>

**Comment by yoannchaudet** [comment]
<p><a class="mention" href="/twiecki">@twiecki</a> I suggest you reach out to Namecheap to ask them where <code>162.255.119.56</code> is coming from. It is currently an <code>A</code> record on your domain (<code>twiecki.io</code>) which is preventing GitHub Pages from requesting a TLS certificate for it.</p>

**Comment by slgraff** [comment]
<p>I expect that IP address may be for a Namecheap parking page. While I’m not familiar with the Namecheap DNS management interface, looking at the following article on their help site it appears that you can disable it from the <code>Other Domain Settings</code> section:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://www.namecheap.com/support/knowledgebase/article.aspx/9250/46/how-can-i-point-my-domain-to-namecheap-parking-page/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/12915136/181077972-b4477aa6-2d99-412f-89f6-bb942c0b85ba.png" class="site-icon" width="16" height="16">

      <a href="https://www.namecheap.com/support/knowledgebase/article.aspx/9250/46/how-can-i-point-my-domain-to-namecheap-parking-page/" target="_blank" rel="noopener">namecheap.com</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:169/48;"><img src="https://user-images.githubusercontent.com/12915136/181077973-92332a26-baff-4159-95ac-122d03f2f14e.png" class="thumbnail" width="169" height="48"></div>

<h3><a href="https://www.namecheap.com/support/knowledgebase/article.aspx/9250/46/how-can-i-point-my-domain-to-namecheap-parking-page/" target="_blank" rel="noopener">How can I point my domain to Namecheap parking page? - Domains - Namecheap.com</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by twiecki** [comment]
<p>I reached out to them but don’t have the sense they know what’s going on. They suggested I remove the CNAME www record, but that doesn’t seem right, does it?</p>
<p><a class="mention" href="/slgraff">@slgraff</a> I did disable it, does that take time to take effect? How can a new cert be issued?</p>

**Comment by twiecki** [comment, marked as answer]
<p>I also removed the www redirect which seems to have fixed it:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/0/101fc2ebdec0fd4654dad007c91187b9d13b18b2.jpeg" data-download-href="/uploads/short-url/2iDGEOE1rJQGJdwCQDjXx7vhN6O.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/674200/181077977-08801ba1-f1a4-437a-9f45-f52ab274ca2b.jpeg" alt="image" data-base62-sha1="2iDGEOE1rJQGJdwCQDjXx7vhN6O" width="620" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/0/101fc2ebdec0fd4654dad007c91187b9d13b18b2_2_620x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/1/0/101fc2ebdec0fd4654dad007c91187b9d13b18b2_2_930x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/0/101fc2ebdec0fd4654dad007c91187b9d13b18b2.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/0/101fc2ebdec0fd4654dad007c91187b9d13b18b2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">980×790 109 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Does a new one get issued automatically?</p>

**Comment by yoannchaudet** [comment]
<p>Visiting the Pages settings page would have been enough to trigger the certificate request. This is where we do a DNS check and queue up certificate renewal/re-provisioning when we detect changes.</p>
<p>I kicked off a renewal manually and it was approved. Within 10 minutes the certificate will be available in the CDN.</p>
<p>Something is funky with Namecheap lately. We do support <code>www</code> variant of certificates nowadays. In any cases your DNS is looking great now.</p>

**Comment by PanZiwei** [comment]
<p>I had a samilar issue with the Github pages. I used to redirect the Github page to a custom domain bought on Namecheap but I didn’t renew it this year. However, the Github page still redirects to the expired domain and I can’t turn on the “Parking Page” on the namecheap since my domain is already expired. Any idea to solve the redirect problem?</p>
